book/version_control.md

```diff
@@ -360,30 +292,39 @@ We can then define our cleaning instructions in a `.gitconfig` configuration fil
 
 If you adjust the filter at any point, you should run this command to apply the changes to existing files:
 
 ```
 git add --renormalize .
 ```
 
-This approach has been borrowed from [this blog post by Yury Zhauniarovich](https://zhauniarovich.com/post/2020/2020-10-clearing-jupyter-output-p3/).
+This approach has been borrowed from this [blog post by Yury Zhauniarovich](https://zhauniarovich.com/post/2020/2020-10-clearing-jupyter-output-p3/).
+
+
+### Access control
+
+Remote version control platforms, like GitHub, have a [visibility setting](https://docs.github.com/en/free-pro-team@latest/github/administering-a-repository/setting-repository-visibility) that represents whether the repo can be viewed publicly or only by owners of the project. We find that some analytical teams choose to develop code in the open, as per guidance from the [Government Service Manual](https://www.gov.uk/service-manual/service-standard/point-12-make-new-source-code-open). Others work in a private repositories, and then might either change to public or migrate to a fresh public repository to make the code open.
+
+```{caution}
+Access control should not be used instead of the good practices outlined above. Restricting access can help to limit the risk of releasing sensitive information, but correct use of version control tools avoids these issues.
+```
 
+On GitHub, Organizations provide an area for collating multiple repos that are associated with a particular team or department. Within Organizations, Teams can be also created to manage view and contribution permissions for specific projects within the Organization. External collaborators can also be added to projects, to allow direct contribution from those outside of the Organization. Detailed setup and management of Organizations and Teams are described in the [relevant section of the GitHub documentation](https://docs.github.com/en/free-pro-team@latest/github/setting-up-and-managing-organizations-and-teams).
 
-### Handling data breaches via Git repositories
+Regardless of the visibility status of a repo, only Organization members and collaborators may make direct contributions to the code in the repo. Others can contribute by Forking and using Pull Requests.
 
-If unauthorised individuals may access sensitive information through their accidental inclusion in a remote repository, this is a data breach.
-For example, pushing credentials or sensitive data to a remote public GitHub repository.
+
+### Handling data breaches via Git
+
+If unauthorised individuals may access sensitive information through their accidental inclusion in a remote repository, this is a data breach. For example, pushing credentials or sensitive data to a remote public GitHub repository.
 
 To handle a data breach, you should:
```

Comment:
It is worth stressing that point about stuff not being gone from the history as that is the killer for people to understand - you can't just delete, commit and go on with your day when it comes to public repos.

Reply:
And as part of that chat worth saying that folloiwng best practices and being conciencious from the get go with not commiting sensitive stuff will help avoid longwinded procedures to clean your history. Also need a link to a guide on how to remove specific files from the history. Something easy to read. 

The guides on that usually end up talking about bfg tool that can help you clean your history but our ONS users won't have ability download java and run that tool to clean the history so maybe find a tutorial explaining how to scrub files from the history that only uses git tools. 

Also worth covering that you shouldn't force push and what happens when git `histories diverge` and how to avoid getting yourself into that position

Reply:
Just saw some of this covered below. Leaving here anyway